guides/source/documents.yaml

```diff
@@ -207,46 +207,46 @@
       url: 6_0_release_notes.html
       description: Release notes for Rails 6.0.
     -
-      name: 5.2 Release Notes
+      name: Version 5.2 - April 2018
       url: 5_2_release_notes.html
       description: Release notes for Rails 5.2.
     -
-      name: 5.1 Release Notes
+      name: Version 5.1 - April 2017
       url: 5_1_release_notes.html
       description: Release notes for Rails 5.1.
     -
-      name: 5.0 Release Notes
+      name: Version 5.0 - June 2016
       url: 5_0_release_notes.html
       description: Release notes for Rails 5.0.
     -
-      name: 4.2 Release Notes
+      name: Version 4.2 - December 2014
       url: 4_2_release_notes.html
       description: Release notes for Rails 4.2.
     -
-      name: 4.1 Release Notes
+      name: Version 4.1 - April 2014
       url: 4_1_release_notes.html
       description: Release notes for Rails 4.1.
     -
-      name: 4.0 Release Notes
+      name: Version 4.0 - June 2013
       url: 4_0_release_notes.html
       description: Release notes for Rails 4.0.
     -
-      name: 3.2 Release Notes
+      name: Version 3.2 - January 2012
       url: 3_2_release_notes.html
       description: Release notes for Rails 3.2.
     -
-      name: 3.1 Release Notes
+      name: Version 3.1 - August 2011
       url: 3_1_release_notes.html
       description: Release notes for Rails 3.1.
     -
-      name: 3.0 Release Notes
+      name: Version 3.0 - August 2010
       url: 3_0_release_notes.html
       description: Release notes for Rails 3.0.
     -
-      name: 2.3 Release Notes
+      name: Version 2.3 - March 2009
       url: 2_3_release_notes.html
       description: Release notes for Rails 2.3.
     -
-      name: 2.2 Release Notes
+      name: Version 2.2 - November 2008
```

Comment:
Are we going by git tag version or rubygem release date? https://github.com/rails/rails/releases/tag/v2.2.0 shows October 2008. Also for 2.3 https://github.com/rails/rails/releases/tag/v2.3.0 is February 2009. I don't see these releases on rubygems 😕

Reply:
Yes, I don't see `2.2.0` and `2.3.0` but later `2.2.x` and `2.3.x` are there. It doesn't look like they were published?

Reply:
I used the blog for these: 
- https://weblog.rubyonrails.org/2008/11/21/rails-2-2-i18n-http-validators-thread-safety-jruby-1-9-compatibility-docs/
- https://weblog.rubyonrails.org/2009/3/16/rails-2-3-templates-engines-rack-metal-much-more/

Not sure what happened to the RubyGems releases.